api/filter/http/transcoder.proto

```diff
@@ -29,7 +29,15 @@ message GrpcJsonTranscoder {
   //   --descriptor_set_out=proto.pb test/proto/bookstore.proto
   //
   // If you have more than one proto source files, you can pass all of them in one command.
-  string proto_descriptor = 1 [(validate.rules).string.min_bytes = 1];
+  oneof {
```

Comment:
please look at the generated docs in CI. I believe ^ is not going to attach properly now (but not sure). You may need to move them somewhere else and then reference them from both places.

Reply:
Done